### ISSUE TITLE
Dot-based groupId don't produce the correct Maven repo structure

### ISSUE THREAD

**Issue by gkgeorgiev** [diffplug/goomph#164]
While using the p2AsMaven gradle plugin, I stumbled upon the following bug:
```
p2AsMaven {

    // the maven group they shall have
    group 'p2.eclipse-plugins', {
        // the repositories and artifacts to download
        p2.addRepo("<URL to P2 repo>")
        iu('<artifact-name>', "${tapVersion}")
    }
 
}
```

Dot-based groupIds result with a wrong folder structure.

It should create the following directory structure:  asmaven > maven > p2 > eclipse-plugins > ......
and the actual result: asmaven > maven > p2.eclipse-plugins > ......

As a result of this, while resolving all classpath dependencies, gradle fails to locate the artifacts.

**Comment by nedtwigg** [comment]
PR's welcome.

**Comment by nedtwigg** [comment]
Fixed in `3.36.1` thanks to @jmini.
